.github/workflows/publish.yml

```diff
@@ -24,6 +24,23 @@ jobs:
         env:
           REF: ${{ github.ref }}
         run: |
+          # we are using ubuntu-latest as our riscv64 cross build machine
+          if [ "${{ matrix.os }}" = ubuntu-latest ]; then
+             APT_INSTALL="sudo apt install -y binfmt-support qemu-user-static"
+             # the following weird statement is here to speed up the happy path
+             # if the default server is responding -- we can skip apt update
+             $APT_INSTALL || { sudo apt update && $APT_INSTALL ; }
+             # we need to build our own version of cross-enabled linuxkit, while upstream is sorting things out
+             make LINUXKIT_VERSION=master LINUXKIT_SOURCE=https://github.com/rvs/linuxkit.git linuxkit
+             # constraining environment for riscv64 builds
+             EVE_BUILDER_IMAGE=$(./build-tools/bin/linuxkit pkg show-tag pkg/alpine)-riscv64
+             echo "PKGS=pkg/alpine pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/mkimage-raw-efi" >> "$GITHUB_ENV"
+             echo "ZARCH=riscv64" >> "$GITHUB_ENV"
+             echo 'LK_BUILD_ARGS={"EVE_BUILDER_IMAGE": "'$EVE_BUILDER_IMAGE'"}' >> "$GITHUB_ENV"
+             echo "EVE_ARTIFACTS=images/docker-compose.yml" >> "$GITHUB_ENV"
```

Comment:
What are all of these vars that we are overriding?

Reply:
Meta question: do you want that answer to be in a doc? In a comment above? @deitch

Reply:
Oh definitely a doc!

Reply:
done